BCStudentSoftwareDevTeam/lsf
# Supervisor search presets

Summary:
Created buttons for potential preset options on the supervisor portal. When a preset option is clicked we create a dictionary based on the presets filtering. With these changes, there are also two new options in the `getDatatableData()` function it will now allow requests for the current user and current term filtering as well as limit non-admins to only information within their department. We have also patched a bug that would attempt to check Tracy first before checking our DB we now have it check our DB first then Tracy and if that fails we pass on the record.

To Test: 
Try each preset and ensure the result match as expected. Also if you click pending forms or current term and click through each page you should see that no result throws an error.
